Makefile

```diff
@@ -0,0 +1,119 @@
+#***************************************************************************
+# set options
+#***************************************************************************
+# set curl configure options
+export configure_build_options=-"-enable-static --disable-shared --disable-ldap --enable-ipv6 --enable-unix-sockets"
+
+# set docker build options used when building docker images
+export docker_build_options=--compress --squash
+
+# set docker build args used when building docker images
+export docker_build_args =--build-arg CURL_CONFIGURE_OPTION=${configure_build_options} \
+    --build-arg CURL_RELEASE_TAG=latest \
+    --build-arg CURL_GIT_REPO=https://github.com/curl/curl.git \
+    --label Name=curl --label Version=1.0.0 \
+    --label se.haxx.curl=curl --label se.haxx.curl.version=1 --label se.haxx.curl.release_tag=1 \
+    --label se.haxx.curl.description="network utility"
+
+#***************************************************************************
+# build docker images
+#***************************************************************************
+.build: .build-centos .build-alpine .build-debian
+.build-centos:
+	cd centos/latest;make build;
```

Comment:
I have reservations about just providing latest; we should also be tagging by release-tag. `latest` is a reasonably weird concept anyway, and basically the only use here would be for someone who just quickly wanted to use `curl` without caring. You can't properly use it in a pipeline without pinning the release number.

Reply:
yes you are right, will change